image-processing/core/configurator.py

```diff
@@ -82,19 +88,55 @@ def __continues_parameter(limits: tuple[any, any], default: any, is_int: bool =
 }
 
 
-def __run(image: str | cv2.typing.MatLike, output: Optional[str], parameters: dict[str, any]) -> cv2.typing.MatLike:
+def __run(
+        image: str | cv2.typing.MatLike | dict[str, np.ndarray],
```

Comment:
cv2.typing.MatLike does unfortunately not work with our conda environment. Maybe we change it to np.ndarray so that it does not complicate things. Otherwise we should at least mention it in a README